hastory-cli/src/Hastory/Cli/OptParse/Types.hs

```diff
@@ -36,13 +36,19 @@ data Flags =
     }
   deriving (Show, Eq)
 
-data Configuration =
-  Configuration
+data Environment =
+  Environment
+    { envBaseUrl :: Maybe T.Text
+    , envCacheDir :: Maybe FilePath
+    , envToken :: Maybe T.Text
+    }
   deriving (Show, Eq)
 
+type RawRemoteStorageClientInfo = (T.Text, T.Text)
```

Comment:
Nit: best to make this a record because now I don't know what these two texts are ..

Reply:
👍